source/_layouts/base.twig

```diff
@@ -24,6 +24,8 @@
     <link rel="preconnect" href="https://ajax.googleapis.com" pr="1.0" />
     <link rel="preconnect" href="https://ssl.google-analytics.com" pr="0.8" />
 
+    <link rel="alternate" type="application/atom+xml" href="{{ site.host }}/atom.xml" title="PHP-FIG Blog Feed" />
```

Comment:
Is `site.host` really needed? All other links are relative, starting with `/` to explicitly point to the root.

Reply:
Nope, removed.